crates/tauri/src/path/android.rs

```diff
@@ -152,6 +152,21 @@ impl<R: Runtime> PathResolver<R> {
     self.call_resolve("getCacheDir")
   }
 
+  /// Returns the path to the app's documents directory.
+  pub fn app_documents_dir(&self) -> Result<PathBuf> {
+    self.call_resolve("getFilesDir")
```

Comment:
Both of these methods call `getFilesDir` and return the same path. What about external files directory on Android? Would this be a better fit for `app_documents_dir`? How is this pattern handled in other platforms like Expo, MAUI etc?